### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by zertosh**
While working on batching DOM ops, these things caught my eye, so I figured I'd factor them out since they're pretty safe and unrelated.

cc: @jerone

### CODE REVIEW

File: lib/tool-bar-view.js [hunk 1]
```diff
@@ -47,19 +45,14 @@ export default class ToolBarView {
         } else {
           this.hide();
         }
+      }),
+      atom.config.onDidChange('tool-bar.fullWidth', ({newValue, oldValue}) => {
```
Comment (jerone) [inline lib/tool-bar-view.js]: Disposable

File: lib/tool-bar-view.js [hunk 2]
```diff
@@ -2,8 +2,6 @@
 
 import {CompositeDisposable, Emitter} from 'atom';
 
-const supportFullWidth = typeof atom.workspace.addHeaderPanel === 'function';
```
Comment (jerone) [inline lib/tool-bar-view.js]: :+1: I'm fine with removing this; it's been there since 1.6.0 Stable

File: lib/tool-bar-view.js (hunk 1 above)
Comment (jerone) [inline lib/tool-bar-view.js]: Disposable

**Comment by jerone** [comment]
One small change, other then that I'm +1 on merging.

**Comment by jerone** [comment]
I merged #162 & #161 which cleaned up some tings too, resulting in (small) merge conflict here.

File: lib/tool-bar-view.js (hunk 1 above)
Comment (zertosh) [inline lib/tool-bar-view.js]: Yeah, I moved it into the giant `this.subscriptions.add` above it. It's just hard to see unless you expand the entire file.

Comment (jerone) [inline lib/tool-bar-view.js]: You're absolutely right.

**Comment by zertosh** [comment]
Rebased and fixed conflicts. Will merge once the tests are done.

**Comment by jerone** [comment]
@cakecatz The change in this PR to rename the private property `toolBar` to `toolBarView` results in an error at `flex-tool-bar`. I noticed [this commit from you](https://github.com/cakecatz/flex-toolbar/commit/42d35de2f874e289edf00b5c01fecdbd6ec686a0) needing that property. There was no issue attached to it, so I was wondering why you need that property? Maybe we can fix it differently.

**Pull request merged**